schema.sql

```diff
@@ -32,4 +32,24 @@ REFERENCES species (id);
 ALTER TABLE animals ADD owner_id INT;
 ALTER TABLE animals
 ADD FOREIGN KEY (owner_id) 
-REFERENCES owners (id);
\ No newline at end of file
+REFERENCES owners (id);
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(100),
+    age INT,
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    vet_id INT REFERENCES vets (id),
+    species_id INT REFERENCES species (id),
+    PRIMARY KEY (vet_id, species_id)
```

Comment:
- [ ] PRIMARY KEY should be something unique and it's one value, not the pair. I highly suggest that you give this `specializations` table and `id` attribute which will be the Primary key, just like you did in other tables 😃 👍